ortools/linear_solver/linear_solver.cc

```diff
@@ -1743,6 +1743,12 @@ bool MPSolver::SupportsCallbacks() const {
   return interface_->SupportsCallbacks();
 }
 
+#if defined(USE_XPRESS)
```

Comment:
why not use SetSolverSPecificParametersAsString ?

Reply:
see: https://github.com/google/or-tools/blob/86d4c543f717fa8716a9d66c25143660668bf825/ortools/linear_solver/linear_solver.h#L648

https://github.com/google/or-tools/blob/86d4c543f717fa8716a9d66c25143660668bf825/ortools/linear_solver/linear_solver.cc#L351-L355

So you could extent `SetSolverSpecificParametersAsString(parameters)`

Reply:
Thanks for the suggestion, I made a commit removing the `SetOptimizationFlag` function and I could use `SetSolverSpecificParametersAsString` to define this flag parameter.